src/ui/server/controllers/wellknown/security/index.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from '../../../../types';
```

Comment:
Can we update the file directory to be `well-known` instead of `welknown` please